### PR TITLE
[PWD-2509] Finalize Navbar

### PR DESCRIPTION
**Pull request by gangsararyop**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

image

image

### CODE REVIEW

File: src/components/Navbar/components/MenuNavbar/index.tsx [hunk 1]
```diff
@@ -15,35 +14,37 @@ interface Menu extends ExtendMenu {
...
-  const menus: Menu[] = useMemo(
-    () => [
-      {
-        value: 6,
-        icon: <Notifications />
-      }
-    ],
-    []
-  )
+  const menus: Menu[] = useMemo(() => [], [])
```
Comment (shadiqfauzi) [inline src/components/Navbar/components/MenuNavbar/index.tsx:17]: kosong gini gan?

Comment (shadiqfauzi) [reply src/components/Navbar/components/MenuNavbar/index.tsx:17]: kalau belum kepake paling kasih notes kalau mau pake ini gimana, dan kasih examplenya jg

Comment (gangsararyop) [reply src/components/Navbar/components/MenuNavbar/index.tsx:17]: sengaja gw kosongin, icon notif blm kepake kan? atau udah? wkwk

Comment (gangsararyop) [reply src/components/Navbar/components/MenuNavbar/index.tsx:17]: ahh okeoke, gw kasih notes aja kalo gtu

**Review by shadiqfauzi — APPROVED** [review]

**Review by Vesals — DISMISSED** [review]

**Review by haikal1223 — APPROVED** [review]

**Review by katherinedavenia — APPROVED** [review]

**Pull request merged**